README.md

```diff
@@ -61,17 +61,28 @@ Postgres in Docker for Windows has a problem accessing the files when they are m
 Just add a hash # in front of the "volumes:" and "- ./docker-mounts/postgres:/var/lib/postgresql/data" line
 
 #### Adjust some DNS settings:
+If you want to use the management-ui add the following lines to your host file.
+This allows you to access container by domain name via your local docker ip address.
  - open "/etc/hosts" as root on Linux / Unix or "C:\Windows\System32\drivers\etc\hosts" on Windows as Administrator
- - add the following lines (and don't forget additional names if you specify more containers!)
+ - and don't forget additional names if you specify more containers!
 
 ```
-127.0.0.1       postgres redis zookeeper kafka kafka-reverse-proxy
-127.0.0.1       flink-jobmanager flink-taskmanager analytics management-ui
-127.0.0.1       marketplace producer consumer merchant-machine-learning
-127.0.0.1       merchant-sample-cheapest merchant-sample-fix-price
-127.0.0.1       merchant-sample-random-third merchant-sample-second-cheapest
-127.0.0.1       merchant-sample-two-bound merchant-simple-competition-logic1
-127.0.0.1       merchant-simple-competition-logic2
+172.29.0.1      postgres redis zookeeper kafka kafka-reverse-proxy
+172.29.0.1      flink-jobmanager flink-taskmanager analytics management-ui
+172.29.0.1      marketplace producer consumer merchant-machine-learning
+172.29.0.1      merchant-sample-cheapest merchant-sample-fix-price
```

Comment:
Just to get that right: Docker will now always use these IPs, no matter where (e.g., Windows, Mac, Linux) the container is started?

Is that IP range a well accepted solution?